src/Build/Dockerfiles/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Argument that takes Algorithm-Image-Name as input
```

Comment:
Should use a folder strcture and/or naming structure to indicate this is the Dockerfile for Gradio interfaces